attacks/01-01-Fingerprint_attack.adoc

```diff
@@ -60,47 +60,98 @@ The evaluator may vary the pressure, angle of rotation, region of the friction r
 
 *Attack Potential Rating Suggestion*
 
-The attack potentials that are required to build the artefacts are summarized in the following table. See BIOSD Section 9 for more information about how to calculate attack potential.
+The attack potentials that are required to build the artefacts are summarized in the following tables. See BIOSD Section 9 for more information about how to calculate attack potential. The <<calculatedtable>> table shows the resulting attack potential based on the highest score for each factor from the mold or cast. The individual attack potential for the mold or cast individually is shown in the tables after. The component column shows where the score comes from.
```

Comment:
Suggest moving this sentence after when mold/cast combos are described (suggestion 1 of 2)

```suggestion
The attack potentials that are required to build the artefacts are summarized in the following tables. See BIOSD Section 9 for more information about how to calculate attack potential.
```